src/components/Tab/TabPanel.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+import { TAB } from '../ThemeIdentifiers';
+import * as baseTheme from './Tab.scss';
+
+/* Position of Tabs in panel
+  // top - tab panel will be shown at a top of content
+  // bottom - tab panel will be shown at a bottom of content
+  // left - tab panel will be shown at a left of content
+  // right - tab panel will be shown at a right of content
+*/
+export type Position = 'top' | 'bottom' | 'left' | 'right';
+
+/* Alignment of tabs in panel
+  // left - tabs alignment starting from left side
+  // center - tabs alignment will be in center
+  // right - tabs alignment starting from right side
+*/
+export type Alignment = 'left' | 'center' | 'right';
+
+// All prototypes type
+export interface Props {
+  // Children are the tabs and it's content
+  children?: any;
+  // Position prop defines the tab's location with reference to it's content
+  position: Position;
+  // Alignment prop defines the tab's Alignment in tab panel
+  alignment: Alignment;
+  // For Default active tab
+  defaulttabid?: string;
```

Comment:
@HiirenP Use camel case for all the keys, so it should be `defaultTabId`

Reply:
I was forced to do this change for only prop because i was getting a warning message something like this. "React does not recognize the `defaultTabId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `defaulttabid` instead".

spent some time to research on this but not found any solution can you please help me in this?

Reply:
This is a weird issue, I will have a look